.github/workflows/cla_bot.yml

```diff
@@ -0,0 +1,24 @@
+name: "CLA Assistant"
```

Comment:
I dont know how to test this but okay to go this in. 

You can potentiall create a new github user and open a PR from fork repo to simulate this.

Reply:
There is an external PR I'd like to test this on, worst case I'll email them the CLA directly

Reply:
once you push this, you can rebase with master and try it out on external PR!